### PR TITLE
Switch brand array to dict

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

File: temba/orgs/models.py [hunk 1]
```diff
@@ -571,7 +572,10 @@ def is_verified(self):
 
     @cached_property
     def branding(self):
-        return brands.get_by_slug(self.brand)
+        return self.get_brand()
+
+    def get_brand(self):
+        return copy.deepcopy(settings.BRAND)
```
Comment (rowanseymour) [inline temba/orgs/models.py:578]: I notice we're always copying brand.. obviously not much downside, but is it really necessary?

File: temba/contacts/tests.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
 from temba.msgs.models import Broadcast, Msg, SystemLabel
 from temba.orgs.models import Org, OrgRole
 from temba.schedules.models import Schedule
+from temba.settings_common import BRAND
```
[the remark this answers is not in the document]
Comment (ericnewcomer) [reply temba/contacts/tests.py:36]: No, mistake!

File: temba/orgs/models.py (hunk 1 above)
Comment (rowanseymour) [reply temba/orgs/models.py:578]: Are we modifying the dict anywhere?

Comment (ericnewcomer) [reply temba/orgs/models.py:578]: I didn't see one when I poked around, but I remember in the recent past where we were causing issues from that happening. I just prefer the failsafe.

Comment (ericnewcomer) [reply temba/orgs/models.py:578]: If you find this super offensive, we can remove it, but the blood will be on your hands!

Comment (rowanseymour) [reply temba/orgs/models.py:578]: I remember when I did some brand tidy up maybe a year ago getting rid of code that modified BRANDS.. and it feels like a little bit of a crutch to just copy everywhere... I don't feel too strongly but I do want us to know whether we're modifying this.

**Pull request merged**